Client/Frontend/Browser/TabManager.swift

```diff
@@ -1145,6 +1145,34 @@ class TabManagerNavDelegate: NSObject, WKNavigationDelegate {
           pref = preference
         })
     }
+    
+    // Mixed-Content validation
```

Comment:
is there any way to add tests for this stuff?

Reply:
Hmm, not exactly sure if we should do it. 
We'd likely have to run a web-view that loads a site with mixed-content forms, and submit the form via JS and monitor the web-view's state.